mdx-models/src/main/java/com/mx/path/model/mdx/model/challenges/Question.java

```diff
@@ -100,6 +102,14 @@ public String getErrorMessage() {
     return errorMessage;
   }
 
+  public void setIsBottomAnchored(Boolean isBottomAnchored) {
+    this.isBottomAnchored = isBottomAnchored;
+  }
+
+  public Boolean getIsBottomAnchored() {
+    return isBottomAnchored;
+  }
```

Comment:
As discussed in Path Learn & Train, let's replace the getters/setters with Lombok generated methods.